<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Participants</class>
 <widget class="qMRMLWidget" name="Participants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>601</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0" colspan="4">
       <widget class="QFrame" name="navigationBar">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="previousPageButton">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>BACK</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="nextPageButton">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>NEXT</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="participantSearchText">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="3" rowspan="3">
       <widget class="QGroupBox" name="optionsGroupBox">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="editParticipantButton">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Edit participant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteParticipantButton">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Delete participant</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_1">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Participant selection</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QTableWidget" name="participantsTable">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Surname</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Birth Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="newParticipantButton">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>New participant</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="newParticipantGroupBox">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>New participant</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="newParticipantLabel_1">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="newParticipantNameWarning">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="newParticipantNameText">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="newParticipantLabel_2">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Surname:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="newParticipantSurnameWarning">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="newParticipantSurnameText">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="newParticipantSaveButton">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="newParticipantLabel_3">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Date of birth:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDateEdit" name="newParticipantBirthDateEdit">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2021</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="newParticipantLabel_4">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Email:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="newParticipantEmailWarning">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="newParticipantEmailText">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QPushButton" name="newParticipantCancelButton">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="4">
          <widget class="QLabel" name="newParticipantWarningMessageText">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ff0000;&quot;&gt;* These fields are required to create a new participant.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QGroupBox" name="editParticipantGroupBox">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Edit participant</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="editParticipantLabel_1">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="editParticipantNameWarning">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="editParticipantNameText">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="editParticipantLabel_2">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Surname:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="editParticipantSurnameWarning">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="editParticipantSurnameText">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="editParticipantSaveButton">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="editParticipantLabel_3">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Date of birth:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDateEdit" name="editParticipantBirthDateEdit">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2021</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="editParticipantLabel_4">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Email:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="editParticipantEmailWarning">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="editParticipantEmailText">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QPushButton" name="editParticipantCancelButton">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="4">
          <widget class="QLabel" name="editParticipantWarningMessageText">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ff0000;&quot;&gt;* These fields cannot be empty.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
